src/components/Playback/Playback.tsx

```diff
@@ -4,7 +4,7 @@ declare var KalturaPlayer: any;
 import "./player.css";
 
 type Props = {
-    media: PlaybackMedia; // the actual recorded media
+    media?: PlaybackMedia; // the actual recorded media
     screenMedia?: PlaybackMedia; // the actual recorded media
```

Comment:
two rows with the same comment.
```suggestion
    cameraMedia?: PlaybackMedia; // the actual recorded camera
    screenMedia?: PlaybackMedia; // the actual recorded screen
```

Reply:
donedone